dist/index.html

```diff
@@ -0,0 +1,36 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <title>Leaderboard</title>
+  <script defer src="/leaderboard/runtime.bundle.js"></script><script defer src="/leaderboard/main.bundle.js"></script></head>
+  <body>
+    <h1>Leaderboard</h1>
+    <div class="all-style">
+      <div class="board">
+        <div class="title-refreshB">
+          <h2>Recent scores</h2>
+          <button type="submit" class="refresh">Refresh</button>
+        </div>
+        <ul>
+          <li>Name: 100</li>
+          <li class="change-bkg">Name: 20</li>
+          <li>Name: 50</li>
+          <li class="change-bkg">Name: 78</li>
+          <li>Name: 125</li>
+          <li class="change-bkg">Name: 77</li>
+          <li>Name: 42</li>
+        </ul>
+      </div>
+      <div class="add">
+        <h2>Add your score</h2>
+        <form>
+          <input type="text" placeholder="Your name" />
+          <input type="number" placeholder="Your score" />
+          <button type="submit" class="score-submit">submit</button>
+        </form>
+      </div>
+    </div>
+  </body>
```

Comment:
- [ ] Kindly position the elements as in the wireframe provided, styling is NOT required, BUT placement of the elements should be followed as per requirement

Provided
![image](https://user-images.githubusercontent.com/81932478/179856084-a4f34ae2-8fd0-493a-a989-5ec9168e81ca.png)

Given
![image](https://user-images.githubusercontent.com/81932478/179856138-b25109ec-ccec-498f-b282-624a735b139d.png)